classify.go

```diff
@@ -29,15 +29,24 @@ type Confidence struct {
 	// The associated confidence with the label.
 	Confidence float32 `json:"confidence"`
 }
+
+type LabelConfidence struct {
+	Confidence float32 `json:"confidence"`
+}
+
 type Classification struct {
-	// The text that is being classified.
-	Input string `json:"input"`
 
-	// The predicted label for the text.
+	// The top predicted label for the text.
 	Prediction string `json:"prediction"`
 
-	// The confidence score for each label.
-	Confidences []Confidence `json:"confidences"`
+	// Confidence score for the top predicted label.
+	Confidence float32 `json:"confidence"`
```

Comment:
why is the top predicted label confidence not available in the node sdk?

Reply:
Good catch! https://github.com/cohere-ai/cohere-node/pull/66